python/nammu/SOAPClient/SOAPClient.py

```diff
@@ -33,13 +37,35 @@ def send(self):
         """
         Elaborate HTTP POST request and send it to ORACC's server.
         """
-        pass
+        url = "http://oracc.museum.upenn.edu:8085"
+        headers = dict(self.request.get_headers())
+        body = self.request.get_body()
+        self.response = requests.post(url, data=body, headers=headers)
+        # try:
+        #    response = requests.post(url, data=body, headers=headers, timeout=10)
+        # except ReadTimeout:
+        #          print "Timed out!"
 
-    def get_response(self):
+    def get_response_text(self):
+        return self.response.text
+
+    def get_response_id(self):
+        xml_root = ET.fromstring(self.response.text)
+        # This should be done with xpath. See XPath and namespaces sections
+        # here: https://docs.python.org/2/library/xml.etree.elementtree.html
+        return xml_root[0][0][0][0].text
+
+    def wait_for_response(self, id):
         """
         Check for a response to the request and obtain response zip file.
         """
-        pass
+        while True:
+            ready_response = requests.get('http://oracc.museum.upenn.edu/p/' + id)
```

Comment:
Server name should probably only be defined once? define server, port and subdir below and combine it here